Chi-squared test updated

Aakriti-G opened the pull request:
Dear @shrabasteebanerjee, referring to PR #975, I am sending a new PR for the chi squared test. Hopefully it is more aligned to what you had in mind. Please let me know what you think! 

Unfortunately, because of exams, I will not be able to get back to it this week but definitely will work on this the week after.
Happy holidays :)

hannesdatta commented:
I have integrated this into our new site structure, so when rendering this branch, you will be able to preview the content.

That said, I still see major quality issues. @Aakriti-G, have you done a grammar/grammarly check? Second, I sense that the original problems pointed out in #975 have not been solved.

@shrabasteebanerjee, can you provide guidance to @Aakriti-G one more time? I will post a link below so you can directly go to the preview site to see the current state. 

Thanks!

hannesdatta commented:
--> https://deploy-preview-1001--unique-scone-f4eb67.netlify.app/topics/analyze/statistical-tests/non-parametric-tests/chi-sq/
--> https://deploy-preview-1001--unique-scone-f4eb67.netlify.app/topics/analyze/statistical-tests/non-parametric-tests/binomial/

Aakriti-G commented:
Hi @hannesdatta Apologies that the new draft still falls short in addressing the issues from previous versions.

I am open to any specific guidance or suggestions you and @shrabasteebanerjee  may have to enhance the overall quality. Thank you for your patience and understanding.

lachlandeer commented:
hi @Aakriti-G @shrabasteebanerjee 

Is this also a obsolete PR after the previous integration? 

Please update accordingly
